assets/aws/files/bin/teleport-generate-config

```diff
@@ -422,6 +422,7 @@ proxy_service:
   tunnel_listen_addr: 0.0.0.0:${TUNNEL_LISTEN_PORT}
   web_listen_addr: 0.0.0.0:3080
   public_addr: ${TELEPORT_EXTERNAL_HOSTNAME:-$PUBLIC_IP}:${TELEPORT_EXTERNAL_PORT:-3080}
+  kube_listen_addr: 0.0.0.0:3026
```

Comment:
This part is already handled in `teleport-generate-config` here: https://github.com/gravitational/teleport/blob/719daf5d9dbfb208b3ad93d9158a2428999f228f/assets/aws/files/bin/teleport-generate-config#L439-L440

https://github.com/gravitational/teleport/blob/719daf5d9dbfb208b3ad93d9158a2428999f228f/assets/aws/files/bin/teleport-generate-config#L84-L106

It uses the older `proxy.kubernetes.listen_addr` format rather than the newer `kube_listen_addr` - this is because when I added it, there wasn't any other way to set `kube_public_addr`. I think support for that was added recently though.

I would just drop this line for now as it will have unpredictable effects. When the config generation script next gets overhauled we can switch to the newer `kube_listen_addr`/`kube_public_addr` format.